crunchy/cli/compare_cmd.py

```diff
@@ -11,19 +11,34 @@
 
 @click.command()
```

Comment:
```suggestion
@click.command("compare-checksums")
```
Would be nice to know what we are comparing